completions/lintian

```diff
@@ -124,8 +124,8 @@ _comp_cmd_lintian()
 
     case "$cur" in
         --*)
-            COMPREPLY=($(compgen -W "$lint_actions $general_opts
-                $behaviour_opts $configuration_opts" -- "$cur"))
+            _comp_compgen -- -W "$lint_actions $general_opts $behaviour_opts
+                $configuration_opts"
```

Comment:
Maybe, per https://github.com/scop/bash-completion/pull/961#discussion_r1186844166
```suggestion
            _comp_compgen -- -W '$lint_actions $general_opts $behaviour_opts
                $configuration_opts'
```

Reply:
Ah, OK. These are also essentially the same as https://github.com/scop/bash-completion/pull/972#discussion_r1188402795. I'd later process them.